src/components/ebay-carousel/utils/scroll-end/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Calls a function every time scrolling completes for an element.
+ * Returns a function to stop listening for scroll ends events.
+ *
+ * (Only works in mobile and relies on touch events)
+ *
+ * @param {HTMLElement} el The element which scrolls.
+ * @param {(offset: number)=>{}} fn The function to call after scrolling completes.
+ * @return {function} A function to cancel the scroll listener.
+ */
+module.exports = function onScrollEnd(el, fn) {
```

Comment:
(minor) You may want to name the file `on-scroll-end` also to clarify the usage, especially as opposed to `scroll-transition` which actually executes scrolling.

Reply:
👍